Assets/Plugins/CountlySDK/Models/CountlyConfiguration.cs

```diff
@@ -58,5 +64,36 @@ public override string ToString()
         {
             return $"{nameof(Salt)}: {Salt}, {nameof(EnablePost)}: {EnablePost}, {nameof(EnableConsoleLogging)}: {EnableConsoleLogging}, {nameof(IgnoreSessionCooldown)}: {IgnoreSessionCooldown}, {nameof(NotificationMode)}: {NotificationMode}, {nameof(EnableManualSessionHandling)}: {EnableManualSessionHandling}, {nameof(SessionDuration)}: {SessionDuration}, {nameof(EventQueueThreshold)}: {EventQueueThreshold}, {nameof(StoredRequestLimit)}: {StoredRequestLimit}, {nameof(TotalBreadcrumbsAllowed)}: {TotalBreadcrumbsAllowed}, {nameof(EnableAutomaticCrashReporting)}: {EnableAutomaticCrashReporting}";
         }
+
+        /// <summary>
+        /// Disabled the location tracking on the Countly server
+        /// </summary>
+        public void DisableLocation()
+        {
+            IsLocationDisabled = true;
+
+            City = null;
```

Comment:
Keep the config file dumb and move the logic to the SDK. Remove these value clearers.

Reply:
location being disables of configuration, so values should be cleared here.

Reply:
done.